lib/platform/linux/platform.c

```diff
@@ -38,6 +38,7 @@ static pthread_t thread_dispatch;
 #if MAX_DURATION_POLL_FAIL_S != 0
 // Last successful poll request
 static time_t m_last_successful_poll_ts;
+static unsigned int max_poll_fail_duration;
```

Comment:
Could you prefix it with m_ as it is a static variable of the module?  and a _s suffix would also help to remember the unit (so m_max_poll_fail_duration_s)